std_msgs/Header header
float64 side_length
float64 voxel_length
int32[] occ_indices
